app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -41,37 +47,45 @@ val level = "Level1"
 fun TopicListRoute(
     modifier: Modifier = Modifier,
     onClickToDetail: (id: String) -> Unit,
+    onClickToMain: () -> Unit,
     viewModel: TopicListViewModel = hiltViewModel()
 ) {
-    TopicListScreen(onClickToDetail = onClickToDetail, viewModel = viewModel)
+    TopicListScreen(
+        onClickToDetail = onClickToDetail, onClickToMain = onClickToMain, viewModel = viewModel
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)